examples/mqtt.eve

```diff
@@ -0,0 +1,21 @@
+
+# MQTT
+
+## Listening to incoming messages
+
+```eve
+search @mqtt
+  m = [#message #incoming topic payload]
+bind @browser
+  [#div text: "MQTT message at {{topic}} : {{payload}}"]
+```
+
+## Sending messages
+
+```eve
+search
+  i = range[from: 1 to: 3]
+
+commit @mqtt
+  [#message #outgoing topic: "eve/test/" + i, payload: i]
```

Comment:
This was just what I instincivally reached for as a first-day Eve programmer, and it worked, so I didn't change it. :D Agreed that string interpolation is preferred, it is easier to understand the code, and it is less magical than implicit string conversions (which I don't think are very nice at all).